README.md

```diff
@@ -51,6 +51,11 @@ for the [Wikidata project](https://wikidata.org/).
 
 ## Release notes
 
+### 0.5 (2015-08-06)
+
+* Drop support for ValueValidators in DataTypes
+* Simplified DataTypeFactory
```

Comment:
I agree, these are not good release notes entries, as they do not tell the users of the component what they need to know. I've updated these lines shortly after merging this PR though.